website/package.json

```diff
@@ -15,18 +15,18 @@
     "archive": "node archive.js"
   },
   "dependencies": {
-    "@docusaurus/core": "2.4.1",
-    "@docusaurus/plugin-client-redirects": "2.4.1",
-    "@docusaurus/preset-classic": "2.4.1",
-    "@mdx-js/react": "1.6.22",
+    "@docusaurus/core": "3.0.0",
+    "@docusaurus/plugin-client-redirects": "3.0.0",
+    "@docusaurus/preset-classic": "3.0.0",
+    "@mdx-js/react": "^3.0.0",
```

Comment:
Dependencies need to be pinned to the correct version. Please remove the `^`

Reply:
Updated, thanks!